### PR TITLE
Update ChartData.swift

### PR DESCRIPTION
**Pull request by mustafamns3a**
Since calcMinMax(dataset:) already calculated max and min for axis. 
(i.e)  _leftAxisMax, _leftAxisMin,  _rightAxisMax, _rightAxisMin.
No need for recalculating.

### Goals :soccer:

Avoiding recalculation.

### CODE REVIEW

**Comment by liuxuan30** [comment]
@jjatie it seems ok to me. I don't remember why it has recalculation this way.

Will it has conflicts for 4.0 branch? or we make this a 4.0 ready PR?

**Comment by jjatie** [comment]
I believe this is already a part of 4.0, or part of a bigger PR for it

**Comment by jjatie** [comment]
Let's target 4.0 though. Everything should be targeting it now.

**Comment by liuxuan30** [comment]
@mustafamns3a please switch to 4.0 branch

File: Source/Charts/Data/Implementations/ChartBaseDataSet.swift [hunk 1]
```diff
@@ -411,13 +421,26 @@ open class ChartBaseDataSet: NSObject, IChartDataSet
...
+        copy.highlightEnabled = highlightEnabled
+        copy._valueFormatter = _valueFormatter
+        copy.valueFont = valueFont
+        copy.form = form
+        copy.formSize = formSize
+        copy.formLineWidth = formLineWidth
+        copy.formLineDashPhase = formLineDashPhase
+        copy.formLineDashLengths = formLineDashLengths
+        copy.drawValuesEnabled = drawValuesEnabled
+        copy.drawValuesEnabled = drawValuesEnabled
```
Comment (xymtek) [inline Source/Charts/Data/Implementations/ChartBaseDataSet.swift]: duplicate line

**Pull request closed without merging**